package.json

```diff
@@ -24,18 +28,27 @@
   "jest": {
     "verbose": true
   },
+  "engines": {
+    "node": ">=8.7.0"
```

Comment:
Is this the Node version you are running?

Reply:
node version was updated to `12.14.0`